features/redirect.feature

```diff
@@ -25,3 +25,9 @@ Feature: Redirect
     And the store has a redirect from path "/old-path" to "/new-path" on channel "United States"
     When I try to access "/old-path"
     Then I should still be on "/old-path"
+
+  Scenario: Redirect from old path to new path and keep query string
```

Comment:
I'd add a few more scenarios, just to make sure we're covering all possible use cases:

* from: /old-path
* to: /new-path#something_in_javascript=thank_you_react

or

* from: /old-path
* to: /new-path?q=ts

so we can have:

/old-path?gclid=1

forwarded to:

/new-path?q=ts&gclid=1

Reply:
I agree with Gabi here. Could we add that scenario